openeducat_admission/models/admission.py

```diff
@@ -462,3 +467,22 @@ class OpStudentCourseInherit(models.Model):
     product_id = fields.Many2one(
         'product.product', 'Course Fees',
         domain=[('type', '=', 'service')], tracking=True)
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = 'res.config.settings'
+
+    global_user = fields.Boolean(config_parameter='global_user_id',
```

Comment:
Parameter name not so relevant, use better name

Reply:
okay sir